<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialogComputers</class>
 <widget class="QDialog" name="dialogComputers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>535</width>
    <height>338</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>535</width>
    <height>338</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Компьютеры</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <widget class="QFrame" name="frameCentral">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item>
       <widget class="QTableWidget" name="tableWidgetComputers">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="tabKeyNavigation">
         <bool>false</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="showGrid">
         <bool>false</bool>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderMinimumSectionSize">
         <number>23</number>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>23</number>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widgetControl" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>276</width>
          <height>0</height>
         </size>
        </property>
        <widget class="QFrame" name="frameInUse">
         <property name="geometry">
          <rect>
           <x>4</x>
           <y>112</y>
           <width>269</width>
           <height>37</height>
          </rect>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="midLineWidth">
          <number>0</number>
         </property>
         <widget class="QLabel" name="label_4">
          <property name="geometry">
           <rect>
            <x>16</x>
            <y>8</y>
            <width>111</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Использование:</string>
          </property>
          <property name="textFormat">
           <enum>Qt::PlainText</enum>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
         <widget class="QPushButton" name="buttonUsed">
          <property name="geometry">
           <rect>
            <x>136</x>
            <y>8</y>
            <width>57</width>
            <height>21</height>
           </rect>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>Да</string>
          </property>
         </widget>
         <widget class="QPushButton" name="buttonNotUsed">
          <property name="geometry">
           <rect>
            <x>198</x>
            <y>8</y>
            <width>57</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Нет</string>
          </property>
         </widget>
        </widget>
        <widget class="QPushButton" name="buttonDel">
         <property name="geometry">
          <rect>
           <x>198</x>
           <y>156</y>
           <width>73</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Удалить</string>
         </property>
        </widget>
        <widget class="QFrame" name="frameColor">
         <property name="geometry">
          <rect>
           <x>4</x>
           <y>68</y>
           <width>269</width>
           <height>37</height>
          </rect>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="midLineWidth">
          <number>0</number>
         </property>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>16</x>
            <y>8</y>
            <width>111</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Цвет маркера:</string>
          </property>
          <property name="textFormat">
           <enum>Qt::PlainText</enum>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
         <widget class="QPushButton" name="buttonNone">
          <property name="geometry">
           <rect>
            <x>137</x>
            <y>8</y>
            <width>22</width>
            <height>22</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QPushButton" name="buttonRed">
          <property name="geometry">
           <rect>
            <x>161</x>
            <y>8</y>
            <width>22</width>
            <height>22</height>
           </rect>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">background-image: url(:/graphics/BtnRed.png);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QPushButton" name="buttonLime">
          <property name="geometry">
           <rect>
            <x>185</x>
            <y>8</y>
            <width>22</width>
            <height>22</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-image: url(:/graphics/BtnGreen.png);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QPushButton" name="buttonAqua">
          <property name="geometry">
           <rect>
            <x>209</x>
            <y>8</y>
            <width>22</width>
            <height>22</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-image: url(:/graphics/BtnAqua.png);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QPushButton" name="buttonYellow">
          <property name="geometry">
           <rect>
            <x>233</x>
            <y>8</y>
            <width>22</width>
            <height>22</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-image: url(:/graphics/BtnYellow.png);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
        <widget class="QPushButton" name="buttonAdd">
         <property name="geometry">
          <rect>
           <x>122</x>
           <y>156</y>
           <width>73</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Добавить</string>
         </property>
        </widget>
        <widget class="QFrame" name="frameAddress">
         <property name="geometry">
          <rect>
           <x>4</x>
           <y>4</y>
           <width>269</width>
           <height>57</height>
          </rect>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="midLineWidth">
          <number>0</number>
         </property>
         <widget class="QLabel" name="labelNumber">
          <property name="geometry">
           <rect>
            <x>16</x>
            <y>7</y>
            <width>81</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Номер:</string>
          </property>
          <property name="textFormat">
           <enum>Qt::PlainText</enum>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
         <widget class="QLabel" name="labelAddress">
          <property name="geometry">
           <rect>
            <x>16</x>
            <y>29</y>
            <width>81</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>IP-адрес:</string>
          </property>
          <property name="textFormat">
           <enum>Qt::PlainText</enum>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEditNumber">
          <property name="geometry">
           <rect>
            <x>104</x>
            <y>6</y>
            <width>45</width>
            <height>21</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEditAddress">
          <property name="geometry">
           <rect>
            <x>104</x>
            <y>30</y>
            <width>157</width>
            <height>21</height>
           </rect>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>28</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>31</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>dialogComputers</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>dialogComputers</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
